llzk-sys/build-support/src/mlir.rs

```diff
@@ -26,20 +26,27 @@ impl<'a> MlirConfig<'a> {
         }
     }
 
-    pub fn mlir_path(&self) -> Result<Cow<'static, Path>> {
-        let path = Path::new(env!("MLIR_SYS_200_PREFIX"));
+    pub fn mlir_path(&self) -> Result<Option<Cow<'static, Path>>> {
```

Comment:
It's been a while since I did any serious rust-ing, but why is this copy-on-write? It seems like the only contexts you use this in treat the path as read-only data.